### PR TITLE
admin-app: Fix wink command

### LINKED ISSUE #9: admin-app: Wink command does not have any effect
Currently, the wink command of the admin app only sets a flag but does not cause a UI change.

### PR DESCRIPTION
**Pull request by robin-nitrokey**
Previously, the wink command did not have any effect.  With this patch,
we toggle the LED between blue and green if the wink command has been
executed.  The LED blinks until the specified duration (currently 10
seconds) is over or the UI status is changed.

This requires a patched trussed version for this change:
	https://github.com/trussed-dev/trussed/pull/15

Fix #9.

### CODE REVIEW

File: runners/lpc55/board/src/trussed.rs [hunk 1]
```diff
@@ -126,10 +129,35 @@ RGB: RgbLed,
...
+        }
     }
 
     fn refresh(&mut self) {
-        if self.rgb.is_some() && self.buttons.is_some() {
+        if self.rgb.is_none() {
+            return;
+        }
+
+        if let Some(wink) = self.wink.clone() {
```
Comment (szszszsz) [inline runners/lpc55/board/src/trussed.rs:144]: Looks good, though I would encapsulate Wink implementation more (separate object)

File: runners/lpc55/board/src/trussed.rs [hunk 2]
```diff
@@ -126,10 +129,35 @@ RGB: RgbLed,
...
+            let time = self.uptime();
+            if wink.contains(&time) {
+                // 250 ms blue, 250 ms green
+                let color = if (time - wink.start).as_millis() % 500 < 250 {
```
Comment (szszszsz) [inline runners/lpc55/board/src/trussed.rs:148]: Can you confirm with @jans23 these are the expected colors?
It matches Nitrokey Storage winking, but not Nitrokey FIDO2 (which has single color white winking).

Comment (robin-nitrokey) [reply runners/lpc55/board/src/trussed.rs:148]: Changed to white blinking.

File: runners/lpc55/board/src/trussed.rs (hunk 1 above)
Comment (robin-nitrokey) [reply runners/lpc55/board/src/trussed.rs:144]: Per our discussion:  As we will have to do some refactoring once we add the NRF chip, we don’t want to over-complicate it at the moment and keep it as-is.

**Comment by robin-nitrokey** [comment]
The admin app has been extracted into a separate repository, so I’ve created a new PR for that part: https://github.com/solokeys/admin-app/pull/1

**Comment by szszszsz** [comment]
Just FYI, mentioned in https://github.com/Nitrokey/nitrokey-3-firmware/pull/11#issuecomment-979945266 is merged.

**Comment by robin-nitrokey** [comment]
Yes, I’m currently working on rebasing this.

**Comment by robin-nitrokey** [comment]
Updated and rebased onto nitrokey-main.  As the content of the patch has not changed, I’m going to merge it without another review.

**Pull request merged**